crates/starknet-server/tests/common/mod.rs

```diff
@@ -1,53 +1,78 @@
-use std::net::TcpListener;
-use std::process::{Child, Command, Stdio};
-use std::{thread, time};
-
-use hyper::client::HttpConnector;
-use hyper::{Client, StatusCode, Uri};
-use thiserror::Error;
-
-#[derive(Error, Debug)]
-pub enum TestError {
-    #[error("Cannot start Devnet")]
-    DevnetNotStartable,
-    #[error("No free ports")]
-    NoFreePorts,
+#[cfg(test)]
```

Comment:
You don't need macro for tests in `tests`. In src its needed so they aren't compiled in `cargo build`. Folder `tests` doesn't participate in compilation

Reply:
Removed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/64/commits/8caa93c60adebb20adc618505dc33da60fa14a8a